ui/src/containers/configs/NewConfig.js

```diff
@@ -304,11 +351,64 @@ class NewConfig extends Component {
     });
   }
 
-  render() {
-    if (this.state.configCreated) {
-      return <Redirect to={"/configs/" + this.props.configs.config.uuid} />;
+  toggleForm(event) {
+    event.preventDefault();
+    let toggle = !this.state.showPayloadEditor;
```

Comment:
`toggle` does not tell a lot. What about renaming it to `isShowingPayloadEditor`?

Reply:
Renamed, thanks for catching this.